CodeHomeWork_2_4/CodeHomeWork_2_4/FileService.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace CodeHomeWork_2_4
+{
+    public class FileService
+    {
+        private static string _path = string.Empty;
+
+        private static string _fileName = string.Empty;
+
+        static FileService()
```

Comment:
Please create FileService with 2 parameters path and fileName. 
Do not use static fields for non-static class in this context, looks strange